### DISCUSSION TITLE
Open.IO ("Insert Diagram") implementation does not support dropdowns in Containers?

### DISCUSSION THREAD

**Discussion by PolyShifter** [requarks/wiki#4229] [Error / Bug Report]
Hey there, in a markdown editor you can use the "Insert Diagram" option to use Open.IO to make a diagram. When you create a container that can be collapsed, and save it, the Container will be missing it's dropdown button.

Is this a bug? Or is there a workaround for this or anything?
Any help is greatly appreciated!

**Comment by Apexy216** [comment]
Also have seen this issue, tried to export it as an HTML diagram and using it on an HTML page but also not working
